client/.gitignore

```diff
@@ -23,4 +23,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # VSCode
-.vscode/
\ No newline at end of file
+.vscode/
+
+# Local Netlify folder
+.netlify
```

Comment:
We don't have a `.netlify` folder.

Reply:
This folder is created during manual deploys (ie. doing a `netlify deploy` from your own terminal). It is usefull for testing purposes.

Reply:
Ok!